owid/datautils/format/numbers.py

```diff
@@ -0,0 +1,305 @@
+"""Numeric formatting."""
+import re
+from typing import Union, Any, Dict, Set
+
+
+class Number:
+    """Wrapper around numbers."""
+
+    def __init__(self, number: Union[str, int]) -> None:
+        self.number_raw = number
+        self.number = self.init_clean(number)
+
+    @classmethod
+    def init_clean(cls, number: Union[int, str]) -> str:
+        """First number cleaning steps.
+
+        Parameters
+        ----------
+        number : Union[int, str]
+            Raw number as given.
+
+        Returns
+        -------
+        int
+            Number as a string, wihtout repeated whitespaces.
+        """
+        # Force string
+        number = num_to_str(number)
+        # Remove multiple spaces
+        number = remove_multiple_whitespaces(number)
+        return number
+
+    def clean(self) -> int:
+        """Clean number."""
+        if self.number.isnumeric():  # already a number
+            return int(self.number)
```

Comment:
What if the number has decimals?

Reply:
I've clarified that this only works for Integer values at the moment